src/common/utxobased/engine/makeUtxoWalletTools.ts

```diff
@@ -1,9 +1,12 @@
 import { AddressPath, CurrencyFormat, NetworkEnum } from '../../plugin/types'
+import { cdsScriptTemplates } from '../keymanager/bitcoincashUtils/checkdatasig'
```

Comment:
Propagate the scriptTemplate in the currency's info (reference the existing bitcoin plugin architecture).

Reply:
I pushed a fixup to implement this.